service/awsconfig/v10/ebs/ebs.go

```diff
@@ -86,24 +86,41 @@ func (e *EBS) DeleteVolume(ctx context.Context, volumeID string) error {
 
 // DetachVolume detaches an EBS volume. If force is specified data loss may occur. If shutdown is
 // specified the instance will be stopped first.
-func (e *EBS) DetachVolume(ctx context.Context, volumeID string, attachment VolumeAttachment, force bool, shutdown bool) error {
-	e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("detaching EBS volume %s from instance %s", volumeID, attachment.InstanceID))
-
+func (e *EBS) DetachVolume(ctx context.Context, volumeID string, attachment VolumeAttachment, force bool, shutdown bool, wait bool) error {
```

Comment:
I am not really happy with the interface because we kind of mix primitives here. I just added something on top for now to keep going. I think once we have to work on this again we should split it up a bit. Not really fuzzed right now though.

Reply:
Good point. Agree this is fine for now. Can be sorted out later.

Reply:
It can be shortened a bit with ` force, shutdown, wait bool`